packages/devtools_app/lib/src/http/http_request_data.dart

```diff
@@ -620,6 +629,25 @@ class DartIOHttpRequestData extends HttpRequestData {
 
   String _responseBody;
 
+  @override
+  String get requestBody {
+    if (_request is! HttpProfileRequest) {
+      return null;
+    }
+    final fullRequest = _request as HttpProfileRequest;
+    try {
+      if (!_request.isResponseComplete) return null;
+      if (_request.method != 'POST') return null;
```

Comment:
should we also be supporting other methods ('CONNECT', 'DELETE', GET', HEAD', PATCH', 'PUT', etc.) @bkonyi

Reply:
I think we should support `PATCH`, `PUT`, and `POST` since those are the request types that have bodies (that I know of, anyway).